documentation/modules/metrics/kafka-exporter/proc_kafka-exporter-configuring.adoc

```diff
@@ -1,90 +0,0 @@
-// Module included in the following assemblies:
-//
-// metrics/assembly_metrics-kafka-exporter.adoc
-
-[id='proc-kafka-exporter-configuring-{context}']
-= Configuring Kafka Exporter
```

Comment:
So, what is this replaced with? Since the content still seems to be relevant.

Reply:
It was incorporated into the [Deploying Prometheus metrics configuration](https://strimzi.io/docs/operators/in-development/deploying.html#proc-metrics-kafka-deploy-options-str) procedure

Reply:
Ok, makes sense. Thanks.